build/psgallery-release.yml

```diff
@@ -88,12 +89,18 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
-          - template: github/create-release.yml@templates
-            parameters:
-              repositoryName: '$(Repository)'
-              releaseNotes: |
-                Install new version via [NuGet](https://www.nuget.org/packages/$(Project)/$(Build.BuildNumber))
-                ```shell
-                PM > Install-Package $(Project) --Version $(Build.BuildNumber)
-                ```
-          - template: nuget/publish-official-package.yml@templates
+          #- template: github/create-release.yml@templates
+          #  parameters:
+          #    repositoryName: '$(Repository)'
+          #    releaseNotes: |
+          #      Install new version via [PowerShell Gallery](https://www.powershellgallery.com/packages?q=$(Project)
+          #      ```shell
+          #      PS > Install-Module -Name $(Project).* --Version $(Build.BuildNumber)
+          #      ```
+          - powershell: |
+              Import-Module PowerShellGet -Force
+              Get-ChildItem -Path ./src -Filter *.psm1 -Recurse |
+                  % { Publish-Module -Path $_.DirectoryName -NuGetApiKey $env:APIKEY -Repository PSGallery -Verbose -Force }
```

Comment:
Instead of `$env:APIKEY`, why are we not using typical variables?

Reply:
I see you pass them as env var but think you can just consume them directly 🤔

Reply:
Yeah, I tried that in the past and I always come back to this bc I couldn't have access to them directly. I think even the official docs explains this in some parts that you have to pass it allong as env vars. Maybe bc it's inline.